styles/styles.css

```diff
@@ -0,0 +1,1423 @@
+@import url("./fonts.css");
+@import url('https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css');
+
+:root {
+  --white: #FFFFFF;
+  --black: #383838;
+  --blue: #3077C6;
+  --yellow: #F5BD41;
+  --grey: #EAF0F9;
+
+  --unit: 8px;
+  --gap: calc(var(--unit) * 2);
+  --container-width: 1232px;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.svg-sprites {
+  position: absolute;
+  visibility: hidden;
+  width: 0;
+  height: 0;
+}
+
+.container {
+  width: 100%;
+  max-width: calc(var(--container-width) + var(--unit) * 4);
+  padding: 0 calc(var(--unit) * 2);
+  margin: 0 auto;
+}
+
+input {
+  border: none;
+}
+
+.phone {
+  display: none;
+}
+
+/*Block 1*/
+.block-one__background {
+  background-image: url("../images/castelmezzano-1979546 1.jpg");
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
+}
+
+.block-one__header {
+  padding-top: 51px;
+  padding-bottom: 202px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.block-one__header--left--svg {
+  width: 205px;
+  height: 40px;
+}
+
+.block-one__header--right {
+  display: flex;
+}
+
+.block-one__header--right--href {
+  margin-right: 24px;
+  font-size: 24px;
+  line-height: 28px;
+  justify-content: space-around;
+  color: #ffffff;
+  text-decoration: none;
+  position: relative;
+  transition: after 300ms;
+}
+
+.block-one__header--right--href::after {
+  pointer-events: none;
+}
+
+.block-one__header--right--href:hover::after {
+  display: block;
+  content: "";
+  height: 2px;
+  width: 91%;
+  background-color: var(--yellow);
+  position: absolute;
+  left: 9%;
+  margin-top: 12px;
+  pointer-events: auto;
+  transition: all 0.8s ease;
+}
+
+.block-one__header--right--logotip {
+  margin-left: 24px;
+  justify-content: space-around;
+  gap: 24px;
+}
+
+.block-one__header--right--logotip:hover {
+  color: #f5bd41;
+}
+
+.block-one__header--right--logotip--night {
+  width: 30px;
+  height: 30px;
+  margin-right: 24px;
+  margin-bottom: 6px;
+}
+
+.block-one__header--right--logotip--account {
+  width: 40px;
+  height: 40px;
+  color: white;
```

Comment:
It is better to choose one format of color definition. Variables, name, hex, what else?

Reply:
yes, ok.